addons/account/models/res_partner_bank.py

```diff
@@ -63,6 +64,22 @@ def _check_allow_out_payment(self):
                 if not self.env.user.has_group('account.group_validate_bank_account'):
                     raise ValidationError(_('You do not have the right to trust or un-trust a bank account.'))
 
+    @api.depends('acc_number')
+    def _compute_alert_duplicate_bank_account(self):
+        """Add a warning if the bank account is already registered with another partner."""
+        duplicate = self.env['res.partner.bank'].search([('acc_number', 'in', self.mapped('acc_number'))], limit=1)
+        for bank in self:
+            bank.duplicate_account_warning = False
+            if not bank.acc_number or not duplicate:
+                continue
+            if bank != duplicate:
+                bank.duplicate_account_warning = _(
+                    "Warning : The Bank Account could be a duplicate of %(name)s (<a href=%(url)s>%(partner)s</a>)",
+                    name=duplicate.display_name,
+                    url=duplicate.partner_id.id,
```

Comment:
This is not a url

You probably want to use `_get_html_link()` instead.

Reply:
You need to manage markup properly, something like
```python
bank.duplicate_account_warning = Markup("%(message)s (%(link)s)") % {
    'message': _(
        "The Bank Account could be a duplicate of %(name)s",
        name=duplicate.display_name,
    ),
    'link': duplicate.partner_id._get_html_link(),
}

Reply:
> Maybe I'm missing something, was your version clickable? (and did it do what was expected?)

yes and yes (redirect to the partner's form)

Reply:
The reason it is working is because the URL is already `.../odoo/customers/xxx`, and you are giving a relative href, which will just replace the `xxx` by your `href`.
So it is not very robust...